src/crypto/secret.ts

```diff
@@ -126,7 +126,7 @@ export function decrypt(
     containsLength?: boolean
 ): Uint8Array {
     let start = 0;
-    let end: number;
+    let end: number | undefined;
```

Comment:
shouldn't this be `let end?: number`? :-P

Reply:
```suggestion
    let end?: number;
```
let's test how suggestions work :)

Reply:
i... think it worked, let's see if ci likes it

Reply:
haha, yeah -- this isn't actually valid syntax here. good to know that accepted suggestions can't actually be reverted, though....

Reply:
(it's for sure confusing that that's allowed for class fields and function parameters but not let-bindings)

Reply:
It kinda makes sense: in class fields and function arguments `name?` means it's optional, that is there might not even be `name` in this context, even though it's no different from it being `undefined`, but `let name` declares that `name` is definitely here, i.e. there's no syntax in JS to make it optional.